src/views/ReadStory.vue

```diff
@@ -70,8 +70,26 @@ async function getStory() {
   }
 }
 
-function deleteStory() {
-  alert('Delete story not implemented yet.')
+async function deleteStory(story, item) {
+
+  // Check if the portion is the parent of the story
+  if (story.parentId == null) {
+    // If it is the parent story, delete all the children stories
+    if(confirm("This will delete the whole story, continue?")) {
+      let i = item.length - 1;
+      while (i > -1) {
+        // Loop through all the stories and delete until there is no more
+        StoryServices.deleteStory(story.userId, item[i].storyId);
```

Comment:
I think this could be accomplished with only the storyid since it is a unique id.